.github/workflows/workflow-generate-ci-documentation.yml

```
@@ -0,0 +1,64 @@
+name: Generate CI Documentation
+on:
+  workflow_call:
+    inputs:
+      commit_resources:
+        description: 'Commit the resources after generating them. (Requires the access_token to be passed)'
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      access_token:
+        description: 'The access token to use for commits.'
+        required: false
+  workflow_dispatch:
+    branches:
+    - main
+    - develop
+    - "feature-*"
+    - "release-*"
+    inputs:
+      commit_resources:
+        description: 'commit the resources after generating them. Requires a PAT defined as secrets.COMMIT_TOKEN'
+        required: true
+        default: false
+        type: boolean
+jobs:
+  update_ci_documentation:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Generate CI Documentation
+        uses: Compton-NIST/Inspector@v1
```

```suggestion
        uses: Compton-NIST/Inspector@a5e3872313458aac125681c7f4d3f2ff2ef6528e
```

Blocking: can we pin to hash of GitHub of `v1` tag, not `v1`? This way we should get updates when new tags emerge from Dependabot. From what I can tell this does not need set up on the producer side (i.e. you for Compton-NIST/Inspector author), just on the consumer side.